### DISCUSSION TITLE
Admin Portal Metadata editing numeric bug

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#38] [Issue]
When adding/editing a metadata value in the admin portal, the text input does not allow the value to start with a number, followed by a space, then an alpha character.

Example:
* key: test
* value: 42 is the answer

When entering the value `"42 is the answer"`, the input changes the value to `"42is the answer"`. Notice the space is stripped away between "42" and "is".

However, you can place the cursor between "42" and "is" after the text has been typed and a space will be accepted.

**Comment by ezekg** [comment]
Thanks for the report. Type inference for metadata values can be finicky. I'll add a test case for this.

But this should now be resolved. May need to reload your dashboard.
